<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfileClass</class>
 <widget class="QMainWindow" name="UserProfileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nunito Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gartic - My Profile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Frontend.qrc">
    <normaloff>:/Frontend/assets/images/appicon.ico</normaloff>:/Frontend/assets/images/appicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-image: url(:/Frontend/assets/images/background.png);
    background-position: center;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_logo">
          <property name="minimumSize">
           <size>
            <width>211</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/gartic-logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_join_game_window">
        <property name="minimumSize">
         <size>
          <width>661</width>
          <height>381</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>661</width>
          <height>381</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Nunito Medium</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="Line" name="split_line">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>20</y>
           <width>2</width>
           <height>341</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(134, 141, 150);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_playnow_badge">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>25</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/profile-badge.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_history_badge">
         <property name="geometry">
          <rect>
           <x>393</x>
           <y>25</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/game-history-badge.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_history_window">
         <property name="geometry">
          <rect>
           <x>358</x>
           <y>90</y>
           <width>278</width>
           <height>271</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 211, 211);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_avatar_background">
         <property name="geometry">
          <rect>
           <x>95</x>
           <y>100</y>
           <width>140</width>
           <height>140</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/letter-avatar-bg.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_user_profile_letter">
         <property name="geometry">
          <rect>
           <x>127</x>
           <y>129</y>
           <width>81</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Nunito Black</family>
           <pointsize>75</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 191, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_name">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Nunito Medium</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>My Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_username">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>279</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Nunito Medium</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(134, 141, 150);</string>
         </property>
         <property name="text">
          <string>@username</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_goBackUserProfile">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>320</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Nunito Medium</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgb(3, 83, 151);
color: rgb(255,255,255);
}

QPushButton:hover, QPushButton:pressed {
   background-color: rgb(2, 63, 112);
}</string>
         </property>
         <property name="text">
          <string> Go Back</string>
         </property>
         <property name="icon">
          <iconset resource="../../Frontend.qrc">
           <normaloff>:/Frontend/assets/images/go-back-ico-white.ico</normaloff>:/Frontend/assets/images/go-back-ico-white.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Frontend.qrc"/>
  <include location="../../Frontend.qrc"/>
 </resources>
 <connections/>
</ui>
